CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing
+
+Welcome. We're really glad you're here. [MuseScore](https://musescore.org/) is exceptional software (yes, we're biased) created, updated and used by many.
+
+It has thorough, organized documentation that contains all the resources you�ll need to help keep MuseScore great. A few of these resources are linked at the bottom. This is a shortened and condensed version to help developers new to MuseScore get started. 
+
+## Getting Started - The Basics
+
+### See MuseScore in action
+* [Install](https://musescore.org/en/handbook/3/installation) it
+* Take a tour (there's an option in the program) or read a little [here](https://musescore.org/en/handbook/3/learning-musescore)
+* Try it out
+
+### Set up your environment
+* Step-by-step directions [here](https://github.com/musescore/MuseScore/wiki/Set-up-developer-environment)
+
+### Get the code
+* Check out the [Suggested Workflow](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/git-workflow#Suggested_workflow)
+* You'll probably want to [fork it](https://www.toolsqa.com/git/difference-between-git-clone-and-git-fork/)
+* Here's the [GitHub repo](https://github.com/musescore/MuseScore)
+
+### Make Updates
+* [Your first contribution](https://musescore.org/en/handbook/developers-handbook/getting-started/your-first-code-contribution)
+* List of all [issues](https://musescore.org/en/project/issues/musescore)
+* Use the [code guidelines](https://github.com/musescore/MuseScore/wiki/CodeGuidelines) (they're _super_ important)
+* [Compile](https://musescore.org/en/handbook/developers-handbook/compilation) the code
+* Continue following the [Suggested Workflow](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/git-workflow) section to keep your branch up-to-date and create a Pull Request
+
+### Contributor Agreement
+* Sign the [CLA](https://musescore.org/en/cla)
```

Comment:
Here you might drop the `en` (and an adjacent `/`), it'll redirect properly.

Reply:
all /3's and /en's dropped from links

Reply:
/en and /3 dropped from links